api/models/user.go

```diff
@@ -208,3 +210,73 @@ func AddItemsToUserInventory(userId primitive.ObjectID, items []interfaces.GymRe
 
 	return nil
 }
+
+// GetLoomiesByUser returns an array of loomies according with user
+func GetLoomiesByUser(loomiesArray []primitive.ObjectID) ([]interfaces.UserLoomiesRes, error) {
+
+	// array to keep loomies ids
+	var loomiesIds []primitive.ObjectID = []primitive.ObjectID{}
+
+	for _, element := range loomiesArray {
```

Comment:
I think this for loop is not necessary, `loomiesArray` is the same that `loomiesIds`.